test/controllers/util/CustomMetadataUtilsSpec.scala

```diff
@@ -106,8 +106,16 @@ class CustomMetadataUtilsSpec extends AnyFlatSpec with MockitoSugar with BeforeA
           field.isInstanceOf[RadioButtonGroupField] should be(true)
           verifyBoolean(field.asInstanceOf[RadioButtonGroupField], property.defaultValue)
         case Text =>
-          field.isInstanceOf[DropdownField] should be(true)
-          verifyText(field.asInstanceOf[DropdownField], property)
+          property.propertyType match {
```

Comment:
This is a duplicate of the block on line 146. Is it worth splitting this into a `checkText` method?